sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/geckoview_version_v1/init.sql

```diff
@@ -0,0 +1,147 @@
+CREATE TABLE IF NOT EXISTS
+  org_mozilla_fenix_derived.geckoview_version_v1(
+    build_hour DATETIME,
+    geckoview_version STRING,
+    n_builds INT64,
+  )
+AS
+WITH extracted AS (
+  -- We'll look at the metrics ping to estimate the major geckoview version.
+  -- The metrics section is aliased, so we must rename the table for this to
+  -- work as expected.
+  SELECT
+    submission_timestamp,
+    client_info.app_build,
+    metrics.string.geckoview_version,
+  FROM
+    org_mozilla_fenix.metrics AS t1
+  WHERE
+    mozfun.norm.fenix_app_info('org_mozilla_fenix', client_info.app_build).channel = 'nightly'
+  UNION ALL
+  SELECT
+    submission_timestamp,
+    client_info.app_build,
+    metrics.string.geckoview_version,
+  FROM
+    org_mozilla_fenix_nightly.metrics AS t1
+  UNION ALL
+  SELECT
+    submission_timestamp,
+    client_info.app_build,
+    metrics.string.geckoview_version,
+  FROM
+    org_mozilla_fennec_aurora.metrics AS t1
+),
+transformed AS (
+  SELECT
+    app_build,
+    geckoview_version,
+    -- Truncate to the hour, since older builds give minute resolution.
+    datetime_TRUNC(mozfun.norm.fenix_build_to_datetime(app_build), HOUR) AS build_hour
+  FROM
+    extracted
+  WHERE
+    DATE(submission_timestamp)
+    BETWEEN DATE_SUB(@submission_date, INTERVAL 30 DAY)
+    AND @submission_date
+),
+grouped_build_hours AS (
+  -- Count the number of geckoview versions for each build hour row over the
+  -- expected interval. We choose a minimum number of builds to filter out
+  -- noise.
+  SELECT
+    build_hour,
+    geckoview_version,
+    COUNT(*) AS n_builds
+  FROM
+    transformed
+  WHERE
+    geckoview_version IS NOT NULL
+    AND app_build IS NOT NULL
+    AND build_hour IS NOT NULL
+  GROUP BY
+    build_hour,
+    geckoview_version
+  HAVING
+    n_builds > 5
+  ORDER BY
+    build_hour DESC,
+    geckoview_version
+),
+top_build_hours AS (
+  -- Get the geckoview version for the build hour that has the most number
+  -- of rows.
+  SELECT
```

Comment:
Subquery is a little clunky but I don't know of a better way.  

Maybe something like
```sql
SELECT
  *,
  ROW_NUMBER() OVER (PARTITION BY build_hour ORDER BY n_builds DESC) AS r,
FROM
  grouped_build_hours t
ORDER BY
  build_hour
```
and filtering by `r=1` in the next query avoids the subquery but I'm not convinced that's any better